glide-core/src/client/value_conversion.rs

```diff
@@ -508,6 +537,7 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
         b"BZMPOP" | b"ZMPOP" => Some(ExpectedReturnType::ZMPopReturnType),
         b"JSON.TOGGLE" => Some(ExpectedReturnType::JsonToggleReturnType),
         b"GEOPOS" => Some(ExpectedReturnType::ArrayOfArraysOfDoubleOrNull),
+        b"LMPOP" => Some(ExpectedReturnType::ArrayOfStringAndArrays),
```

Comment:
Add test to new conversion type
Test manually with RESP2 - use a patch 

```diff
diff --git a/java/client/src/main/java/glide/managers/ConnectionManager.java b/java/client/src/main/java/glide/managers/ConnectionManager.java
index d9a8f585..16e7f2ea 100644
--- a/java/client/src/main/java/glide/managers/ConnectionManager.java
+++ b/java/client/src/main/java/glide/managers/ConnectionManager.java
@@ -93,6 +93,8 @@ public class ConnectionManager {
                             .build());
         }

+        connectionRequestBuilder.setProtocolValue(1);
+
         connectionRequestBuilder
                 .setTlsMode(configuration.isUseTLS() ? TlsMode.SecureTls : TlsMode.NoTls)
                 .setReadFrom(mapReadFromEnum(configuration.getReadFrom()));
```

Reply:
created new branch to clean up git commit history, will address comments on new pr